src/components/Request/NewRequestForm.Wizard.tsx

```diff
@@ -172,40 +190,64 @@ const Wizard = ({ isLoading = false, isError = false }) => {
       {state.page === "USAJobs" && <USAJobs />}
       {state.page === "DONE" && <Done gotoStep={gotoStep} />}
 
-      {!isLoading && (
-        <div className="requestWizardButtons">
+      <div className="requestWizardButtons">
+        <Button
+          disabled={state.page === "RoutingInfo" || addRequest.isLoading}
+          onClick={() => {
+            if (isValid) {
+              dispatch({ type: "prev_page", payload: methods });
+            } else {
+              trigger(undefined, { shouldFocus: true });
+            }
+          }}
+        >
+          Prev
+        </Button>
+        {state.page !== "DONE" && (
           <Button
-            disabled={state.page === "RoutingInfo" || isLoading}
-            onClick={() => {
+            appearance="primary"
+            style={{ marginLeft: "auto" }}
+            disabled={addRequest.isLoading}
+            onClick={async () => {
               if (isValid) {
-                dispatch({ type: "prev_page", payload: methods });
+                await createNewRequest().then(() =>
```

Comment:
With the new "Save and Continue" button -- if they use "Prev" -- it will not save the changes they made, and their only option to save it, is to move forward, then back twice.